### PR TITLE
Pushing updated strings for localization

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

New translation strings

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - Localization

##### AWX VERSION
```
devel
```

### CODE REVIEW

File: awx/ui/src/locales/translations/fr/messages.po [hunk 1]
```diff
@@ -0,0 +1,10713 @@
...
+#: components/PromptDetail/PromptJobTemplateDetail.js:156
+#: components/Schedule/ScheduleDetail/ScheduleDetail.js:489
+#: components/Schedule/ScheduleToggle/ScheduleToggle.js:57
+#: screens/Inventory/SmartInventoryHostDetail/SmartInventoryHostDetail.js:45
+#: screens/Setting/shared/SettingDetail.js:98
+#: screens/Setting/shared/SharedFields.js:149
+#: screens/Template/JobTemplateDetail/JobTemplateDetail.js:284
+#: screens/Template/shared/JobTemplateForm.js:504
+msgid "On"
+msgstr "Le"
```
Comment (AlexSCorey) [inline awx/ui/src/locales/translations/fr/messages.po]: Based on [this PR](https://github.com/ansible/awx/pull/13048/files) the following change should occur:
```suggestion
msgstr "Activé"
```

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
Just 1 small change based on community member contribution.

File: awx/ui/src/locales/translations/fr/messages.po (hunk 1 above)
Comment (rooftopcellist) [reply awx/ui/src/locales/translations/fr/messages.po]: @AlexSCorey I have incorporated that change in this PR and requested that the translators add it to the translation memory.

**Review by AlexSCorey — APPROVED** [review]
Looks good

**Pull request merged**